### ISSUE TITLE
App crashes during long distance travel

### ISSUE THREAD

**Issue by pravusjif** [decentraland/unity-renderer#612]
@nearnshaw has reported a crash when doing all the tram trip all the way and back at:
https://play.decentraland.org/?CATALYST=peer-testing.decentraland.org&position=0%2C12&realm=testing-amber

```
Could not allocate memory: System out of memory!
Trying to allocate: 71663616B with 16 alignment. MemoryLabel: NewDelete
Allocation happened at: Line:0 in Overloaded New[]
```

![Screen Shot 2021-06-15 at 10.31.39.png](https://images.zenhubusercontent.com/5d9200f4f58e25000108abac/f583032e-d44d-4b81-af6f-7fce1e86e1ad)

**Comment by pravusjif** [comment]
I ran a quick test from Unity Editor (had to hack the fast-moving-platforms-failsafe to avoid getting dropped off the wagon all the time) and got the following errors:

![Screen Shot 2021-06-15 at 11.58.18.png](https://images.zenhubusercontent.com/5d9200f4f58e25000108abac/3101ace8-b5f7-474c-beea-ab511372e7f0)

![Screen Shot 2021-06-15 at 11.51.00.png](https://images.zenhubusercontent.com/5d9200f4f58e25000108abac/981de6d2-6120-49b2-83ec-91e610125972)

Regarding memory, what I saw using Unity Profiler is that Garbage Memory was always between 350mb~450mb, and using >2GB total memory constantly.

In Unity i didn't get any crash nor crash screen in the browser. We should debug this also with a build.
